model/project.go

```diff
@@ -1450,20 +1472,56 @@ func (p *Project) BuildProjectTVPairs(patchDoc *patch.Patch, alias string) {
 // mapping of the build variant to the tasks that will run on that build
 // variant. If includeDeps is set, it will also resolve task dependencies.
 func (p *Project) ResolvePatchVTs(patchDoc *patch.Patch, requester, alias string, includeDeps bool) (resolvedBVs []string, resolvedTasks []string, vts []patch.VariantTasks) {
```

Comment:
I would write a test for this function since it's exported and seems to be untested.

Either that, or add tests to `BuildProjectTVPairs` for the new patch regex functionality.

Reply:
wrote testing